<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnCSCDialog</class>
 <widget class="QDialog" name="LearnCSCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>9</number>
   </property>
   <widget class="QWidget" name="Main_page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>951</width>
       <height>521</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>941</width>
       <height>521</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="backgroundPic_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>951</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string>IMAGE</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>771</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="Pointers_btn_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>POINTERS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Vectors_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>VECTORS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="FileInOut_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>FILE I/O</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Array_btn_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>ARRAY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Execption_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>EXCEPTION</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="template_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>TEMPLATE</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Inheritance_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>INHERITANCE</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Recursion_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>RECURSION</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="Class_btn_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noteworthy</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/img/background3_Pic.jpeg);
color: purple;
border-width: 1px;
border-color: purple;
border-style: outset;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>CLASS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="GoBack_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Pointers_page">
    <widget class="QPlainTextEdit" name="Pointers_txt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>521</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/DefaultBackgrd.jpg);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GoBack_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Vector_page">
    <widget class="QPlainTextEdit" name="Vector_txt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>521</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/DefaultBackgrd.jpg);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GoBack_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1 vec</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FileInOut_page">
    <widget class="QPushButton" name="GoBack_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Array_page">
    <widget class="QPushButton" name="GoBack_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Exception_page">
    <widget class="QPushButton" name="GoBack_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Execption_txt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>521</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/DefaultBackgrd.jpg);</string>
     </property>
    </widget>
    <zorder>Execption_txt</zorder>
    <zorder>GoBack_9</zorder>
   </widget>
   <widget class="QWidget" name="Template_page">
    <widget class="QPushButton" name="GoBack_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Inheritance_pae">
    <widget class="QPushButton" name="GoBack_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Recursion_page">
    <widget class="QPushButton" name="GoBack_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Class_button">
    <widget class="QPushButton" name="GoBack_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/blackBacground.jpeg);
color: rgb(255, 147, 0);
font: 900 17pt &quot;Avenir&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
